cli/azd/extensions/microsoft.azd.ai.builder/internal/cmd/start.go

```diff
@@ -789,7 +746,35 @@ func ensureAzureContext(
 	return azureContext, project, nil
 }
 
-func (a *startAction) createQuestions() map[string]qna.Question {
+func (a *startAction) createQuestions(ctx context.Context) (map[string]qna.Question, error) {
+	azdClient, err := azdext.NewAzdClient()
```

Comment:
I believe we already have an azdClient that we can leverage on the action instead of creating a new one here.